EIPS/eip-1.md

```diff
@@ -252,6 +252,24 @@ Permitted Networking Specifications URLs must anchor to a specific commit, and s
 ^https://github.com/ethereum/devp2p/blob/[0-9a-f]{40}/.*$
 ```
 
+### Bitcoin Improvement Proposal
+
+Links to the Bitcoin Improvement Proposal may be included using normal markdown syntax, such as:
+
+```markdown
+[BIP-0038](https://github.com/bitcoin/bips/blob/3db736243cd01389a4dfd98738204df1856dc5b9/bip-0038.mediawiki)
+```
+
+Which renders to:
+
+[BIP-0038](https://github.com/bitcoin/bips/blob/3db736243cd01389a4dfd98738204df1856dc5b9/bip-0038.mediawiki)
+
+Permitted Bitcoin Improvement Proposal URLs must anchor to a specific commit, and so must match this regular expression:
+
+```regex
+^(https://github.com/bitcoin/bips/blob/[0-9a-f]{40}/.*|https://github.com/bitcoin/bips/tree/[0-9a-f]{40}/.*)$
+```
+
```

Comment:
```suggestion

Additionally, links to BIPs must use the hyphenated form `BIP-X`, where `X` is the BIP number.

```

Reply:
BIPs seem to be referenced without hyphen within other BIPs. We should defer to their style of references, no?

Reply:
If that is the BIP style, then we should follow it.